src/components/index/TransactionsList.tsx

```diff
@@ -2,135 +2,89 @@ import { Transaction } from '@defichain/whale-api-client/dist/api/transactions'
   return (
-    <div className='w-full flex flex-wrap p-4 rounded border border-gray-200 cursor-pointer my-1.5 hover:shadow-md'>
-      <div className='w-1/2 lg:w-2/5 xl:w-3/5 flex space-x-2'>
-        <span className='text-lg leading-6'>
-          <MdSwapHorizontalCircle className='text-gray-400 inline-block' size={22} />
-        </span>
-        <div className='overflow-ellipsis overflow-hidden'>
-          <TxIdLink
-            txid={props.txid}
-            className='overflow-ellipsis overflow-hidden font-medium text-gray-900'
-          />
-          <div className='text-xs text-gray-400 leading-5'>
-            <span>{props.age}</span>
-          </div>
-        </div>
-      </div>
-      <DesktopTransactionDetails totalVoutValue={props.totalVoutValue} />
-      <MobileTransactionDetails totalVoutValue={props.totalVoutValue} />
-    </div>
+    <>
+      {props.transactions.map(transaction => {
+        return (
+          <Link href={{ pathname: `/transactions/${transaction.id}` }} key={transaction.hash}>
+            <span className='content'>
+              <TransactionDetails
+                txid={transaction.txid}
+                age={formatDistanceToNow(transaction.block.medianTime * 1000, { addSuffix: true })}
+                totalVoutValue={transaction.totalVoutValue}
+              />
+            </span>
+          </Link>
+        )
+      })}
+    </>
   )
 }
 
-function DesktopTransactionDetails (props: { totalVoutValue: string }): JSX.Element {
+function TransactionDetails (props: { txid: string, age: string, totalVoutValue: string }): JSX.Element {
   return (
-    <div className='hidden md:block w-1/2 lg:w-3/5 xl:w-2/5'>
-      <div className='w-full flex'>
-        <div className='w-1/2 text-right text-sm text-gray-500 lg:mr-2 xl:mr-0'>
-          Amount
+    <div className='flex p-4 rounded border border-gray-200 my-1.5 hover:shadow-md'>
+      <div className='flex-none w-11/12 lg:w-18/19'>
+        <div className='flex'>
+          <div className='flex-none w-8'>
+            <MdSwapHorizontalCircle className='text-gray-400 inline-block' size={22} />
+          </div>
+          <div className='flex-none w-28 md:w-36 xl:w-96'>
+            <div className='overflow-ellipsis overflow-hidden'>
+              <TxIdLink txid={props.txid} className='overflow-ellipsis overflow-hidden font-medium text-gray-900' />
+            </div>
+            <div className='hidden md:block text-xs text-gray-400'>
+              <span>{props.age}</span>
+            </div>
+          </div>
+          <div className='flex-none w-9' />
+          <div className='flex-grow w-10'>
+            <div className='hidden md:block text-sm'>
+              <span className='text-gray-400'>Amount&nbsp;</span>
+              <span>{`${new BigNumber(props.totalVoutValue).toFixed(8)} DFI`}</span>
+            </div>
+            <div className='md:hidden text-center text-xs text-gray-400'>
+              <span>{props.age}</span>
+            </div>
```

Comment:
Try to get the values more aligned to the right. I think you could reuse what was done before just make the texts closer. Similarly for Blocks too.

![image](https://user-images.githubusercontent.com/4586209/145754511-21ccf6b6-d0e6-4caf-ae3b-3b9866e5fae3.png)

Reply:
Ok, noted

Reply:
@joeldavidw I tried to reuse that code but I saw code repeated and hard to reason about, specially in the layout. I suggest to use the new version.

Reply:
May I know the resolution you used?

Reply:
Is it OK for you like this alignment?

<img width="1220" alt="Screenshot 2021-12-15 at 2 57 13 AM" src="https://user-images.githubusercontent.com/2662097/146175801-d4352753-17e4-4c77-b77c-2ca757f704f4.png">

Reply:
Is it OK for you this alignment?

<img width="957" alt="Screenshot 2021-12-15 at 7 07 14 PM" src="https://user-images.githubusercontent.com/2662097/146175876-f545a6f7-dfa6-4a66-b888-44dbe36a9c5e.png">